### PR TITLE
Scala Steward PRs 5670 - 5679 [BA-6558]

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -71,24 +71,26 @@ object Dependencies {
...
   private val scalaPoolV = "0.4.3"
   private val scalacheckV = "1.14.0"
-  private val scalacticV = "3.0.5"
+  private val scalacticV = "3.0.8"
   private val scalameterV = "0.10.1"
-  private val scalamockV = "4.1.0"
+  private val scalamockV = "4.4.0"
   private val scalatestV = "3.0.5"
-  private val scalazV = "7.2.27"
+  private val scalazV = "7.2.30"
```
Comment (cjllanwarne) [inline project/Dependencies.scala:88]: TIL we still import scalaz!??

File: services/src/test/scala/cromwell/services/database/DatabaseSystem.scala [hunk 2]
```diff
@@ -13,13 +13,13 @@ sealed trait DatabaseSystem {
...
-    HsqldbDatabaseSystem,
-    MariadbEarliestDatabaseSystem,
-    MariadbLatestDatabaseSystem,
-    MysqlEarliestDatabaseSystem,
-    MysqlLatestDatabaseSystem,
+//    HsqldbDatabaseSystem,
+//    MariadbEarliestDatabaseSystem,
+//    MariadbLatestDatabaseSystem,
+//    MysqlEarliestDatabaseSystem,
+//    MysqlLatestDatabaseSystem,
```
Comment (cjllanwarne) [inline services/src/test/scala/cromwell/services/database/DatabaseSystem.scala:20]: Pretty sure we don't want this change to be immortalized...

Comment (mcovarr) [reply services/src/test/scala/cromwell/services/database/DatabaseSystem.scala:20]: omg no

File: project/Dependencies.scala (hunk 1 above)
Comment (mcovarr) [reply project/Dependencies.scala:88]: and it's still being updated, it's actually up to 7.4 now

**Comment by cjllanwarne** [comment]
Also, please rebase to check against novel import collisions

File: project/Dependencies.scala [hunk 3]
```diff
@@ -71,24 +71,26 @@ object Dependencies {
   private val owlApiV = "5.1.16"
   private val paradiseV = "2.1.1"
   private val pegdownV = "1.6.0"
-  private val postgresV = "42.2.5"
+  // For org.postgresql:postgresql 42.2.6 - 42.2.14:
+  // java.lang.NoSuchFieldException: m_mesgParts in KeyValueSpec "fail if one of the inserts fails"
+  private val postgresV = "42.2.5" // scala-steward:off
```
Comment (kshakir) [inline project/Dependencies.scala:76]: ToL: I suspect this is an issue with the Cromwell test that should be filed in Jira and investigated at a later date.

Comment (cjllanwarne) [reply project/Dependencies.scala:76]: Created as https://broadworkbench.atlassian.net/browse/BA-6557

**Pull request merged**